src/components/Footer.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="container-fluid footer p-0 page_404">
+    <div class= "row footer justify-content-center m-0">
+      <div class="col-3 footer">
+        <a href ="https://www.cornelldti.org" class= "footer"> Cornell DTI @ {{getYear()}}</a>
+      </div>
+      <div class="col-3">
+        <p class= "footer">Built with LOVE</p>
```

Comment:
Is there a reason we're not using the heart emoji anymore?

Reply:
I looked at the designs and realized they didn't have it - don't know if they ever did or it got updated